### ISSUE TITLE
Rescan blockchain - stuck V1 txes

### ISSUE THREAD

**Issue by Azazel020** [oxen-io/oxen-core#810]
I still have a balance in my wallet that I can`t use. Problem is known by the team but still not fixed.

Below the Discord conversation about this bug:

**Haafingar (Simon)afgelopen woensdag om 01:05**
I dont think I did say it would be fixed, I said it may have had something to do with an unscheduled soft fork?
And besides, I am the wrong person to ask, I am a lowly non-developer. If Jason (jagerman)  and Doyle  have seen your issue, they are the ones most likely to resolve it
 
**Jason (jagerman)afgelopen woensdag om 01:10**
Actually it sort of does involve you Simon; we need to allocate a small amount to pay out to SNs that have stuck amounts (either Rangeproof or LF should do this), and then we will destroy the stuck amounts at the next fork and add a one-time payment to compensate for the payouts.  It just got delayed because of the hectic dev time around the fork, plus various LF activity that was keeping us busy.
Basically Doyle and I looked at the amount of work involved to handle it automatically and decided that it isn't worth bothering giving the relatively small amounts involved; even if we just pay out from LF funds and eat it it would be a better deal that solving it with code.  We just have to be totally above board on it, though.
@Haafingar (Simon)
 
**Haafingar (Simon)afgelopen woensdag om 01:14**
I haven't been keeping abreast of the situation at all, this seems like a reasonable step forward
 
**Jason (jagerman)afgelopen woensdag om 01:19**
We basically need to rescan the blockchain to find all the service node payments that got stuck in v1 txes.  I'm thinking it isn't very many.
But it's an annoyance for the people who have one stuck there.

**Comment by doyle** [comment]
You folks keep pinging me, but you're pinging the wrong doyle. I don't work on this project.
